web/models/stripe_account.ex

```diff
@@ -0,0 +1,38 @@
+defmodule CodeCorps.StripeAccount do
+  @moduledoc """
+  Represents a StripeAccount stored on Code Corps.
+  """
+
+  use CodeCorps.Web, :model
+
+  @primary_key {:id, :string, []} # set primary key as string
+  @derive {Phoenix.Param, key: :id} # set method of determining primary key for route
+  schema "stripe_account" do
+    field :business_name, :string
```

Comment:
We are attempting to simply clone the Stripe API object here for now, or, that is, clone the fields that might be relevant to us. The Stripe object's information needs to be valid for the Stripe API. Conceivably, the organization name/title used on the site, as part of the Organization record might end up being changeable in some way.